src/http-api/fetch.ts

```diff
@@ -274,7 +279,10 @@ export class FetchHttpApi<O extends IHttpOpts> {
                 credentials: "omit", // we send credentials via headers
                 keepalive: keepAlive,
             });
+
+            logger.debug(`FetchHttpApi:  <-- ${res.status} ${Date.now() - start}ms ${urlForLogs}`);
         } catch (e) {
+            logger.debug(`FetchHttpApi:  <-- ${e} ${Date.now() - start}ms ${urlForLogs}`);
```

Comment:
For consistency with the start line:

```suggestion
            logger.debug(`FetchHttpApi: <-- ${res.status} ${Date.now() - start}ms ${method} ${urlForLogs}`);
        } catch (e) {
            logger.debug(`FetchHttpApi: <-- ${e} ${Date.now() - start}ms ${method} ${urlForLogs}`);
```

Reply:
should probably put ${method} also